### PR TITLE
Release version 8.1.2

### PR DESCRIPTION
**Pull request by mcornaton**

### CODE REVIEW

File: src/test/java/com/iexec/core/worker/WorkerServiceRealRepositoryTests.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
+package com.iexec.core.worker;
```
Comment (jbern0rd) [inline src/test/java/com/iexec/core/worker/WorkerServiceRealRepositoryTests.java:1]: License is missing here

File: gradle.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version=8.1.1
+version=8.1.2
 iexecCommonVersion=8.2.1
 iexecCommonsPocoVersion=3.0.4
```
Comment (jbern0rd) [inline gradle.properties:3]: Wouldn't we want to propagate latest iexec-commons-poco here ?

Comment (mcornaton) [reply gradle.properties]: Thanks for the idea. Please see [fc19f08](https://github.com/iExecBlockchainComputing/iexec-core/pull/602/commits/fc19f088388adcbf07a275e156792ac9088a657e).

File: src/test/java/com/iexec/core/worker/WorkerServiceRealRepositoryTests.java (hunk 1 above)
Comment (mcornaton) [reply src/test/java/com/iexec/core/worker/WorkerServiceRealRepositoryTests.java:17]: Good catch! Please see [d36d49f](https://github.com/iExecBlockchainComputing/iexec-core/pull/602/commits/d36d49fe1a931b514fadcc96b45ab91a861e6e2a).

**Review by jbern0rd — APPROVED** [review]
Thanks for this PR and the last fixes !

**Pull request merged**